google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/startup/ConflictingAppEnginePluginCheck.java

```diff
@@ -91,22 +90,20 @@ private void notifyUser(@NotNull Project project, @NotNull IdeaPluginDescriptor
             GctBundle.message("plugin.conflict.error.title"),
             errorMessage,
             NotificationType.ERROR,
-            new IdeaAppEnginePluginLinkListener(project, plugin))
-        .notify(project);
+            new IdeaAppEnginePluginLinkListener(plugin))
+        .notify(null);
```

Comment:
i just meant adding a comment next to the null with the name of the parameter that notify takes so that its clearer to read. Just a convention we've been using to get around Java's lack of named parameters.